spec/unit/puppet_x/chocolatey/chocolatey_install_spec.rb

```diff
@@ -50,3 +50,19 @@
   end
 
 end
+
+describe 'Chocolatey Temp Directory' do
+  context 'on Windows', :if => Puppet::Util::Platform.windows? do
```

Comment:
nit: should we do "skip ('Not on Windows platform') unless Puppet::Util::Platform.windows?" to prevent branching around unit tests?

Reply:
🤔 This is a good question. That's not how we _have_ done things to this point but re our recent question "what tests are actually running where?" it seems appropriate.

Changing now.